### PR TITLE
Fix strip_properties for nested objects. Expand the test case.

### LINKED ISSUE #3: strip_properties does not work on nested objects
Parameter `strip_properties` of `dump` function works only for top-level object. `@property` of nested objects are always included in json. See my gist example:

https://gist.github.com/haluzpav/b578dad460cb3670d55305dc65d9d2bf

Is there a way to also strip properties of nested objects?

### PR DESCRIPTION
**Pull request by haluzpav**
Fixes #3.

This is my first pull request on GitHub. :)

### CODE REVIEW

File: jsons/serializers.py [hunk 1]
```diff
@@ -164,6 +164,7 @@ def default_object_serializer(obj: object,
     """
     obj_dict = obj.__dict__ if strip_properties and hasattr(obj, '__dict__') \
         else _get_dict_from_obj(obj, strip_privates, **kwargs)
+    kwargs['strip_properties'] = strip_properties  # add for possibly nested objects
```
Comment (ramonhagenaars) [inline jsons/serializers.py]: To be more in line with the rest of this function, I'd suggest to just add strip_properties to the return. Like so:

```python
return default_dict_serializer(obj_dict, 
                               key_transformer=key_transformer,
                               strip_nulls=strip_nulls,
                               strip_privates=strip_privates, 
                               strip_properties=strip_properties,
                               **kwargs)
```

**Comment by ramonhagenaars** [comment]
Can you also increase the version number in `setup.py` from `0.5.4` to `0.5.5`?

File: jsons/serializers.py (hunk 1 above)
Comment (haluzpav) [reply jsons/serializers.py]: Oh right, I forgot that Python puts them into kwargs automatically like that. Will do later.

**Comment by haluzpav** [comment]
Sorry about the force pushes, I needed to change commit email.

**Comment by ramonhagenaars** [comment]
This pull request has been superseded by [this one](https://github.com/ramonhagenaars/jsons/pull/7).

**Pull request closed without merging**